atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -155,6 +163,12 @@ private void cas(Client client, Long oldVal, long newVal) {
         }
     }
 
+    private String getCurrentValues(CASResult result) {
+        List<Long> timestamps = result.current_values.stream().map(col -> PtBytes.toLong(col.getValue())).collect(
+                Collectors.toList());
```

Comment:
Can we add the new lines just before `.map` and `.collect`? So it looks like (include the `Collectors.joining` change):
```java
return result.current_values.stream()
        .map(col -> PtBytes.toLong(col.getValue()))
        .collect(Collectors.joining(", "));
```
or maybe:
```java
return result.current_values.stream()
        .map(Column::getValue)
        .map(PtBytes::toLong)
        .collect(Collectors.joining(", "));
```